activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -799,6 +799,16 @@ def rename_column(table_name, column_name, new_column_name)
       #
       # Note: Partial indexes are only supported for PostgreSQL and SQLite.
       #
+      # ====== Creating an index that includes additional columns
+      #
+      #   add_index(:accounts, :branch_id,  include: :party_id)
+      #
+      # generates:
+      #
+      #   CREATE UNIQUE INDEX index_accounts_on_branch_id ON accounts USING btree(branch_id) INCLUDE (party_id)
+      #
+      # Note: only supported by PostgreSQL with btree indexes.
```

Comment:
`UNIQUE` is extra.

> with btree indexes

That was in 11 version, now also `GIST` and `SP-GIST` (https://www.postgresql.org/docs/current/sql-createindex.html). I would remove these extra words: `Note: only supported by PostgreSQL`

Reply:
Good point. I've updated the wording.

Reply:
You forgot to remove `UNIQUE.`